src/main/java/be/seeseemelk/mockbukkit/entity/EndermanMock.java

```diff
@@ -0,0 +1,90 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import be.seeseemelk.mockbukkit.UnimplementedOperationException;
+import com.google.common.base.Preconditions;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.entity.Enderman;
+import org.bukkit.material.MaterialData;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+public class EndermanMock extends MonsterMock implements Enderman
+{
+
+	private MaterialData carriedMaterial = null;
+	private BlockData carriedBlock = null;
```

Comment:
There shouldn't be two separate fields for material and block. There should only be one BlockState that the material methods get the type of, and set to from `BlockStateMock#mock`